core/ledger/pvtdatastorage/store_impl_test.go

```diff
@@ -829,24 +825,6 @@ func TestRollBack(t *testing.T) {
 	assert.True(testDataKeyExists(t, store, datakeyTx0))
 	assert.True(testDataKeyExists(t, store, datakeyTx5))
 	assert.True(testMissingDataKeyExists(t, store, eligibleMissingdatakey))
-
-	// rollback last prepared block and test store for absence of datakeys and eligibile missingdatakeys
-	store.Rollback()
-	testPendingBatch(false, assert, store)
-	testLastCommittedBlockHeight(10, assert, store)
-	assert.False(testDataKeyExists(t, store, datakeyTx0))
-	assert.False(testDataKeyExists(t, store, datakeyTx5))
-	assert.False(testMissingDataKeyExists(t, store, eligibleMissingdatakey))
-
-	// For previously committed blocks the datakeys and eligibile missingdatakeys should still be present
-	for i := 1; i <= 9; i++ {
-		datakeyTx0.blkNum = uint64(i)
-		datakeyTx5.blkNum = uint64(i)
-		eligibleMissingdatakey.blkNum = uint64(i)
-		assert.True(testDataKeyExists(t, store, datakeyTx0))
-		assert.True(testDataKeyExists(t, store, datakeyTx5))
-		assert.True(testMissingDataKeyExists(t, store, eligibleMissingdatakey))
-	}
 }
 
 // TODO Add tests for simulating a crash between calls `Prepare` and `Commit`/`Rollback` - [FAB-13099]
```

Comment:
Forgot to remove `Rollback` from the comment?

Reply:
Done. In fact, some of these discrepancies I believe are present in master as well.